evm/deploy-amplifier-gateway.js

```diff
@@ -77,13 +77,13 @@ async function getDomainSeparator(config, chain, options) {
     }
```

Comment:
also needs to be fixed here
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/cosmwasm/deploy-contract.js#L204